python/storage/test/test_storage.py

```diff
@@ -22,9 +22,9 @@
 
 import pytest
 
-from kserve.storage import Storage
+from storage import Storage
 
-STORAGE_MODULE = "kserve.storage.storage"
+STORAGE_MODULE = "storage.storage"
```

Comment:
I think some comments already proposed this but I just added here again. 
`storage.storage` does not clear to me so why don't we use something like `kserve-storage.storage`